site/docs/components/autocomplete.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Autocomplete
+navTitle: Autocomplete (not built)
+summaryParagraph: Autocomplete automatically completes typed user input with matching results from a larger data set. Autosuggest breaks beyond the input provided to suggest alternative, relevant answers.
+tags: ["Autosuggest", "Live filter", "Filters", "Search", "Top bar", "Text input"]
+needToKnow:
+- 'We separate Search Field into a separate component, because it may be used for live-filtering without showing autocompleted results.'
+- 'Unlike a Select (single- or multi-) that lets you select options from an options menu, the Autocomplete component has other behavior beyond choosing an option as the value for a setting, such as linking to other content.'
+
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## To keep in mind
+
+You might use the **guidance** around autocompletion in other UI or components, such as using good defaults in [Select](/components/select). You might also use the Autocomplete component to present autocomplete options in a dropdown container for [Search Field](/components/search-field).
+
+Using Autocomplete, a user can start writing a partial query and available results are surfaced, which they can select. This works well when the user has some idea of what might be available.
+
+Autocomplete is forgiving, letting people make mistakes such as spelling errors and still predict the content they want.
+
+### Anatomy
+
+- [Dropdown container](/components/dropdown/): the bit that drops down from a [Search Field](/components/search-field) to show Autocomplete options.
+- **Autocomplete options or items:** the options menu that lets you choose options.
+    - Each option/item may be presented as plain text or with rich media, such as images.
+
+### Behavior
```

Comment:
Do we have any guidelines around when the autocomplete kicks in, i.e., whether it is immediate (keyup on the first character) or delayed until some minimum condition (x characters, more than one match etc.). This might be framed more as a consideration related to the search type/dataset/performance.

Reply:
+1 to above!